core/common/src/main/java/alluxio/collections/FieldIndex.java

```diff
@@ -46,6 +47,14 @@
   boolean contains(Object value);
```

Comment:
this naming is now confusing. Maybe this should be called `containsField` and the parameter should be `fieldValue` ?